Assets/Scripts/Tetromino/TetrominoBehavior.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using UnityEngine.Networking;
+
+public class TetrominoBehavior : NetworkBehaviour
+{
+
+  // Use this for initialization
+  public void Start()
+  {
+    if (isServer)
+    {
+      FindObjectOfType<GameMasterBehavior>().RegisterTetromino(this);
+    }
+  }
+
+  // Update is called once per frame
+  public void Update () {
+		
+	}
+
+  public void Move(Direction direction)
+  {
+    switch(direction)
+    {
+      case Direction.Up:

```

Comment:
Up? That'll make tetris a lot easier.

For debugging?

Reply:
No reason for the tetromino to know it can't move up. The PlayerBehavior just doesn't have an "up" command to ever send it.